js/components/app.js

```diff
@@ -9,30 +9,36 @@ ScreenFeedsAndCreatePost
 window.App = React.createClass({
   getInitialState: function () {
     return {
-      route: "GroupDetails"
+      route: "GroupList"
     };
   },
-
   setRoute(r) {
     this.setState({ route: r });
   },
-
-
+  gettContacts: function () {
+    return this.props.jsonContacts;
```

Comment:
Why pass the function `getContacts`? Just pass the `jsonContacts` directly as props to the components. This works but feels like an unnecessary level of indirection. All you are doing is wrapping the `jsonContacts` in function and calling the fn later to extract the value. So, my point is just pass the value itself.